jwst/combine_1d/combine_1d_step.py

```diff
@@ -17,17 +18,17 @@ class Combine1dStep(Step):
     spec = """
     # integration_time or exposure_time.
     exptime_key = string(default="exposure_time")
-    # Interpolation between pixels.
-    interpolation = string(default="nearest")
+    # This is currently not used.
```

Comment:
The `spec` ConfigSpec block needs to have the comment on the same line as the configuration parameter.  Like this:
```
param = string(default="foo") # Comment string
```
otherwise, the comment string doesn't get pulled into the configspec.  Currently, this step's `ConfigSpec` looks like:
```
In [12]: pprint(step.load_spec_file())                                                                                                                                   
{'background': 'boolean(default=False)',
 'exptime_key': 'string(default="exposure_time")',
 'input_dir': 'string(default=None)        # Input directory',
 'interpolation': 'string(default="nearest")',
 'output_dir': 'string(default=None)        # Directory path for output files',
 'output_ext': "string(default='.fits')     # Default type of output",
 'output_file': 'output_file(default=None)   # File to save output to.',
 'output_use_index': 'boolean(default=True)       # Append index.',
 'output_use_model': 'boolean(default=False)      # When saving use '
                     '`DataModel.meta.filename`',
 'post_hooks': 'string_list(default=list())',
 'pre_hooks': 'string_list(default=list())',
 'save_results': 'boolean(default=False)      # Force save results',
 'search_output_file': 'boolean(default=True)       # Use outputfile define in '
                       'parent step',
 'skip': 'boolean(default=False)      # Skip this step',
 'suffix': 'string(default=None)        # Default suffix for output files'}
```